packages/generators/app/src/utils/usage.ts

```diff
@@ -12,43 +11,6 @@ function addPackageJsonStrapiMetadata(metadata: Record<string, unknown>, scope:
   return _.defaults(metadata, packageJsonStrapi);
 }
 
-export async function captureException(error: Error) {
-  try {
-    sentry.captureException(error);
-    await sentry.flush();
-  } catch (err) {
-    /** ignore errors */
-    return Promise.resolve();
-  }
-}
-
-async function captureError(message: string) {
-  try {
-    sentry.captureMessage(message, 'error');
```

Comment:
Are we removing sentry events completely? Or did I miss them being included somehere else?

Reply:
yes :) we don't use it anymore